.github/workflows/staging.yml

```diff
@@ -76,6 +76,8 @@ jobs:
           docker buildx build --platform linux/arm64/v8,linux/amd64 -t $ECR_REGISTRY/akto-billing:$IMAGE_TAG . --push
           cd ../internal
           docker buildx build --platform linux/arm64/v8,linux/amd64 -t $ECR_REGISTRY/akto-internal:$IMAGE_TAG . --push
+          cd ../api-runtime
+          docker buildx build --platform linux/arm64/v8,linux/amd64 -t $ECR_REGISTRY/akto-api-security-runtime:$IMAGE_TAG . --push
```

Comment:
We don't need this na. We don't need to build runtime for akto-repo.

Reply:
Needed this for testing, will remove it before merging